### DISCUSSION TITLE
Ical from existing model

### DISCUSSION THREAD

**Discussion by James-Jefferies** [spatie/icalendar-generator#90] [Q&A]
Hi there,
             I think I'm missing a bit of knowledge of how this works overall.
             
   I would like to share Laravel calendar events with external providers such as outlook or google via the ical url, what I'm not entirely clear on is the workflow. 
   
   Can I create/update an ical file from events stored inside a Database in Laravel? as it's not entirely clear in the examples provided. 
   
   thanks

**Comment by cweiske** [comment]
Yes, this is possible.
You have to write the code that fetches the event data from database yourself, though.
